client/my-sites/email/gsuite-add-users/index.jsx

```diff
@@ -310,7 +311,7 @@ class GSuiteAddUsers extends Component {
 GSuiteAddUsers.propTypes = {
 	currentRoute: PropTypes.string,
 	domains: PropTypes.array.isRequired,
-	googleMailProduct: PropTypes.object.isRequired,
+	googleMailProduct: PropTypes.object,
 	gsuiteUsers: PropTypes.array,
 	isRequestingDomains: PropTypes.bool.isRequired,
 	productType: PropTypes.oneOf( [ GOOGLE_WORKSPACE_PRODUCT_TYPE, GSUITE_PRODUCT_TYPE ] ),
```

Comment:
It looks like we're removing `productType` as a prop, so we should probably remove the `propType` here.
```suggestion
```